src/Sulu/Bundle/AdminBundle/Resources/js/components/Icon/Icon.js

```diff
@@ -28,6 +28,10 @@ export default class Icon extends React.PureComponent<Props> {
         const {className, name, onClick} = this.props;
         let fontClass = '';
 
+        if (!name || name.length <= 0) {
+            log.warn('Invalid icon given: ' + name);
```

Comment:
Isn't there an error like this already somewhere? Thought I have already seen this from time to time.